src/flash.sol

```diff
@@ -117,14 +118,17 @@ contract DssFlash is IERC3156FlashLender, IVatDaiFlashLender {
             return 0;
         }
     }
+
     function flashFee(
         address token,
         uint256 amount
     ) external override view returns (uint256) {
+        amount;
```

Comment:
You can just exclude the parameter name in the function definition.

Reply:
Yes, but that will remove it from the automatic generated abi. So considered it was better this way for this specific case.